plugins/modules/aci_l3out_logical_node.py

```diff
@@ -52,9 +54,20 @@
   loopback_address:
     description:
     - The loopback IP address.
+    - The BGP-EVPN loopback IP address for Infra SR-MPLS L3Outs.
     - A configured loopback address can be removed by passing an empty string (see Examples).
     type: str
     aliases: [ loopback ]
+  mpls_transport_loopback_address:
+    description:
+    - The MPLS transport loopback IP address for Infra SR-MPLS L3Outs.
+    type: str
+    aliases: [ mpls_transport_loopback ]
+  sid:
+    description:
+    - The MPLS transport loopback IP address for Infra SR-MPLS L3Outs.
```

Comment:
change the description to describe what is the `sid` attribute here.  it's currently a duplicate of `mpls_transport_loopback_address`'s description.

Reply:
ok will change